client/image_tag.go

```diff
@@ -12,10 +12,15 @@ import (
 )
 
 // ImageTag tags an image in the docker host
-func (cli *Client) ImageTag(ctx context.Context, imageID, ref string) error {
-	distributionRef, err := distreference.ParseNamed(ref)
+func (cli *Client) ImageTag(ctx context.Context, source, target string) error {
+	_, err := distreference.ParseNamed(source)
 	if err != nil {
-		return fmt.Errorf("Error parsing reference: %q is not a valid repository/tag", ref)
+		return fmt.Errorf("Error parsing reference: %q is not a valid repository/tag", source)
```

Comment:
You're welcome! Sorry for the extra work, but while we're at it, it's nice to do the extra improvements :heart: